CONTRIBUTING.md

```diff
@@ -33,46 +36,53 @@ Since the Closure library i18n data is itself auto-generated from the data of th
 [Common Locale Data Repository (CLDR)] project, errors in the data should
 be reported there. See also the [Closure guide to i18n changes].
 
-**Please see the Submission Guidelines below**.
+**Please see the [Submission Guidelines](#submit) below.**
 
 ## <a name="feature"></a> Want a Feature?
+
 You can request a new feature by submitting an issue to our [GitHub Repository][github].  If you
 would like to implement a new feature then consider what kind of change it is:
 
 * **Major Changes** that you wish to contribute to the project should be discussed first on our
-[dev mailing list][angular-dev] or [IRC][irc] so that we can better coordinate our efforts, prevent
-duplication of work, and help you to craft the change so that it is successfully accepted into the
-project.
-* **Small Changes** can be crafted and submitted to the [GitHub Repository][github] as a Pull Request.
+  [dev mailing list][angular-dev] or [IRC][irc] so that we can better coordinate our efforts,
+  prevent duplication of work, and help you to craft the change so that it is successfully accepted
+  into the project.
+* **Small Changes** can be crafted and submitted to the [GitHub Repository][github] as a Pull
+  Request.
 
 
 ## <a name="docs"></a> Want a Doc Fix?
+
 If you want to help improve the docs, it's a good idea to let others know what you're working on to
-minimize duplication of effort. Before starting, check out the issue queue for
-[Milestone:Docs Only](https://github.com/angular/angular.js/issues?milestone=24&state=open).
```

Comment:
This milestone has been closed. There are no issues on it and we don't use it for new issues.